companies/basecamp.json

```diff
@@ -0,0 +1,35 @@
+{
+    "slug": "basecamp",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Basecamp LLC",
+    "runs": [
+        "Basecamp",
+        "HEY",
+        "Highrise",
+        "Campfire",
+        "Backpack",
+        "Ta-da List",
+        "Writeboard"
+    ],
+    "address": "2045 W Grand Ave Ste B\nPMB 53289\nChicago IL 60612\nUnited States",
+    "email": "privacy@basecamp.com",
+    "web": "https://basecamp.com",
+    "sources": [
+        "https://basecamp.com/about/policies/privacy",
+        "https://basecamp.com/about/policies/privacy/regulations"
+    ],
+    "required-elements": [
```

Comment:
When trying to sign up for an account, I am immediately asked for my name, so I think it makes sense to include that as a required element.